kaue/jsonexport
# Bug: CLI incorrectly parsing JSON Object as JSON Array

## Status
**READY**

## Description
CLI incorrectly parses JSON Object as a JSON Array, leading to a non vertical output (key/value pairs) and instead places keys in header. https://github.com/kaue/jsonexport/issues/91

This PR removes the _wrapArray function which was added in 2017 and parses the chunk directly.

## Steps to Test or Reproduce
Create a file in the main dir `complex.json` using the code below, run `npm install` then `npm run build:dist` to build lib, then run `./bin/jsonexport.js complex.json` to see stdOut

```
{
  "complex": {
    "object": 1,
    "that": 2
  },
  "has": 3,
  "nested": {
    "keys": true
  }
}
```

## Impacted Areas in Application

* CLI
